parquet-avro/README.md

```diff
@@ -0,0 +1,44 @@
+<!--
```

Comment:
Should we add it using separate jira?

Reply:
Usually we separate jiras in similar cases to not make cherry-picking hard in case of the related changed needs to be on another branch as well. In this case this is only a documentation so should not cause any troubles. It would be cleaner if this documentation would have already been existed and I've had to add the docs of the new keys only (which would clearly be part of this change).
If you have a strong opinion to separate this to another change I'm happy to do so, though.

Reply:
Sounds good